docs/source/community/contributing.rst

```diff
@@ -2,100 +2,95 @@
+  Note that this command does not produce output on success
 
 **Pull Request Guidelines**
 
-Pull requests can contain a single commit or multiple commits. The most
-important part is that **a single commit maps to a single fix or enhancement**.
-
-Here are a few scenarios:
+A pull request can contain a single commit or multiple commits. The most
+important guideline is that a single commit should map to a single fix or
+enhancement. Here are some example scenarios:
 
-* If a pull request adds a feature but also fixes two bugs, then the pull
-  request should have three commits, one commit each for the feature and two
-  bug fixes
-* If a PR is opened with 5 commits that was work involved to fix a single issue,
-  it should be rebased to a single commit
-* If a PR is opened with 5 commits, with the first three to fix one issue and
-  the second two to fix a separate issue, then it should be rebased to two
-  commits, one for each issue
+* If a pull request adds a feature but also fixes two bugs, the pull
+  request should have three commits: one commit for the feature change and
+  two commits for the bug fixes.
+* If a PR is opened with five commits that contain changes to fix a single
+  issue, the PR should be rebased to a single commit.
+* If a PR is opened with several commits, where the first commits fix one issue
+  and the rest fix a separate issue, the PR should be rebased to two
+  commits (one for each issue).
 
-Your pull request should be rebased against the current master branch. Please do
-not merge the current master branch in with your topic branch, nor use the
-Update Branch button provided by GitHub on the pull request page.
+**Important:**
+  Your pull request should be rebased against the current master branch. Do
+  not merge the current master branch in with your topic branch. Do not use the
+  Update Branch button provided by GitHub on the pull request page.
 
 **Commit Messages**
 
-Commit messages should follow common Git conventions, such as the imperative
-mood, separate subject lines, and a line-length of 72 characters. These rules
-are well documented by Chris Beam in
-`his blog post <https://chris.beams.io/posts/git-commit/#seven-rules>`_ on the
-subject.
+Commit messages should follow common Git conventions, such as using the
+imperative mood, separate subject lines, and a line length of 72 characters.
+These rules are well documented in `Chris Beam's blog post
+<https://chris.beams.io/posts/git-commit/#seven-rules>`_.
 
 **Signed-off-by**
 
-Commits must include Signed-off-by in the commit message (``git commit -s``).
-This indicates that you agree the commit satisfies the
-`Developer Certificate of Origin (DCO) <http://developercertificate.org/>`_.
+Commits must include the "Signed-off-by" line in the commit message
+(``git commit -s``). This sign-off indicates that you agree the commit satisfies
+the `Developer Certificate of Origin (DCO) <http://developercertificate.org/>`_.
```

Comment:
Could you also please add some verbiage about the git user.email matching the github account. I think this is the identity factor that the system checks.
See https://help.github.com/articles/setting-your-commit-email-address-in-git/